baseapp-chats/baseapp_chats/graphql/mutations.py

```diff
@@ -82,6 +84,26 @@ def mutate_and_get_payload(cls, root, info, profile_id, participants, **input):
                 ]
             )
 
+        if len(participants) > 2 and not is_group:
```

Comment:
Couldn't you just say that `is_group` is true when `len(participans) > 2`, maybe just do
```python
is_group = is_group or len(participants) > 2
```

Reply:
oh thats nice i'll do it thanks